packages/devices/src/getDevices.ts

```diff
@@ -60,6 +60,64 @@ export const getDefaultOsVersion = (platform: Platform): string => {
     throw new Error(`No osVersion for platform: ${platform}`);
 };
 
+export const getFonts = (platform: Platform): Font[] => {
```

Comment:
Can `getFonts` and `getLocations` be moved to separate files? It would be good to separate the location JSON data into individual files as well since there's so much data

ex:
`getFonts.ts` should contain all font related logic,
`getLocations.ts` should contain all location related logic, `locations.json` should contain location data